eatin-client/src/components/hooks/useGetRecipesComments.ts

```diff
@@ -0,0 +1,73 @@
+import { useCallback, useEffect, useState } from "react";
+import {
+    useGetCommentsByRecipeIndexQuery,
+    useCreateCommentsMutation,
+} from "../../generated/graphql";
+import { useGetUsersName } from "./useGetUsersName";
+import { v4 as uuidv4 } from "uuid";
+import { Comments } from "../types";
+
+export function useGetRecipesComments(recipeIndex: number) {
+    const userID = useGetUsersName();
+    const [insertNewComment] = useCreateCommentsMutation();
+    const [isLoading, setIsLoading] = useState(false);
+    const { data: comments, refetch } = useGetCommentsByRecipeIndexQuery({
+        variables: { recipeID: recipeIndex },
```

Comment:
the recipe id is it's index? the index where? in the carousel?

Reply:
Its the ID of the recipe in the DB

Reply:
I know the ID is for the db, I asked what is the index, index where?